openedx/core/djangoapps/schedules/management/commands/__init__.py

```diff
@@ -27,10 +31,18 @@ def add_arguments(self, parser):
             help='Send all emails to this address instead of the actual recipient'
         )
         parser.add_argument('site_domain_name')
+        parser.add_argument(
+            '--duration',
+            type=int,
+            help='The duration for which weekly emails are sent (days).',
+        )
```

Comment:
I think this interface has the potential to confuse users. For instance, if the user puts "21" here, they probably mean three weeks, but will get two. But if we're always going in steps of 7 anyway, then we might as well make the parameter something like "--weeks  = Number of weekly reminder emails to send before giving up." and do the multiplication for them.

Reply:
👍  to this suggestion , thanks @ormsbee !  We'll update this PR in the next couple of weeks, and ping you when it's ready again.  CC @pkulkark